js/packages/core-base/src/types.ts

```diff
@@ -247,16 +249,28 @@ export interface ExtractTextSettings<TRegion = Region> extends ImageSettings<TRe
   userCommandId?: string
 }
 
+export type SelfHealingReport = {
+  operations: [
+    {
+      timestamp: string,
+      old: string,
+      new: string
+    }
+  ]
```

Comment:
This is still not what you want. Here is
```suggestion
  operations: {timestamp: string; old: string; new: string}[]
```

Reply:
Ah 👏